src/main/java/edu/harvard/iq/dataverse/TermsOfUseAndAccess.java

```diff
@@ -287,11 +289,25 @@ private void clearCustomTermsVariables(){
         disclaimer = null;
     }
     
+    @Transient
+    private String validationMessage;
+
+    public String getValidationMessage() {
+        return validationMessage;
+    }
+
+    public void setValidationMessage(String validationMessage) {
+        this.validationMessage = validationMessage;
+    }
+    
     /**
      * @todo What does the GUI use for a default license? What does the "native"
      * API use? See also https://github.com/IQSS/dataverse/issues/1385
      */
-
+    /*
```

Comment:
does this belong here?

Reply:
Remnant of the multiple license merge conflict. removing.